incubator/virtualcluster/pkg/syncer/conversion/helper.go

```diff
@@ -145,3 +145,11 @@ func MutatePod(namespace string, pod *corev1.Pod) {
 	}
 	pod.Spec.Volumes = volumes
 }
+
+func MutateService(newService *corev1.Service) {
+	newService.Spec.Selector = nil
```

Comment:
I thought we should keep the selector from service in tenant master?

Reply:
yeap, we can keep the selector, since service selector works in namespace scope.